internal/csi-common/utils.go

```diff
@@ -33,6 +33,15 @@ import (
 	"k8s.io/klog"
 )
 
+// enum defining logging levels.
```

Comment:
duplicating this should be prevented, move it to a shared package

Reply:
I duplicated to each package for better readability.. i.e klog.V(Default) in place of klog.V(util.Default). As suggested, I will now have a single declaration in utils. Thanks.